data/tutorial/architecture.md

```diff
@@ -29,6 +37,28 @@ the same branch is an atomic and concurrent operation.
 
 Branches are stored in the reference store as pairs of `(hash(commit), branch)`.
 
+As for the block store, you can choose a reference store from the ones provided
+by Irmin, or you can create your own using the interface
+[Irmin.ATOMIC_WRITE_STORE]. The operations you have to implement need to
+satifisy some constraints:
+
+- concurrent reads of the same branch name return the same hash, that is the
```

Comment:
```suggestion
- concurrent reads of the same branch name return the same commit hash
```
I think this clarification only introduces confusion :slightly_smiling_face: The two things are not the same.